source/localizable/feedspec/validator.html.md.erb

```diff
@@ -0,0 +1,9 @@
+---
+parent: feedspec
+title: validator
+---
+
+# Data Feed Validator <%= edit_link %>
+
+You can validate your XML feed according to <%= settings.site_name.capitalize %> standards
+using our public <a href="http://<%= settings.feed_validator_domain %>">Data Feed Validator</a>.
```

Comment:
My suggestion. The feedspec pages are already flavored. No need to add more code and complexity here.

We could add a note for the validator at the [top of the current page](http://developer.skroutz.gr/feedspec/) and be done with it. What do you think?